code/game/machinery/doors/airlock.dm

```diff
@@ -648,6 +648,7 @@
 		else
 			. += "There's a [note.name] pinned to the front..."
 			. += note.examine(user)
+		. += span_notice("The [note.name] can be <b>cut</b> off.")
```

Comment:
```suggestion
		. += span_notice("The [note.name] can be [EXAMINE_HINT("cut")] off.")
```

Reply:
Oh there's a macro for this now? Nice, I guess if I'm ever feeling bored I'll look for examine hints still using <b> (I know there's some)